DataStructures/LinkedList/LinkedList.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+    Author:
+    Tucker J. Polomik (https://github.com/TheNergaL)
+    Name of algorithm:
+    Singly Linked-list
+    Brief description:
+    List comprised of nodes which hold a value
+    and point to the next value element in the list.
+    If there exists no next element, it points to null.
+*/
+
+#ifndef LINKED_LIST_HPP
+#define LINKED_LIST_HPP
+
+template<class T>
+struct Node
+{
+  T value;
+  Node<T> *nextnode;
+};
+
+template<class T>
+class LinkedList
+{
+        Node<T>* head;
+        Node<T>* last;
+        int list_size;

```

Comment:
Rename `list_size` to just `size`.
Also, indentation is more than 1 level (4 spaces) here.